src/nlp/multiple-precision.jl

```diff
@@ -1,21 +1,31 @@
 export multiple_precision_nlp
 
+function multiple_precision_nlp(
+  problem :: String;
+  kwargs...
+)
+  @warn("This function signature will be deprecated, see the help for the new signature")
```

Comment:
Can we use `Base.depwarn` instead of the macro `warn`? (same below)

Reply:
Done